docs/resources/access_group.md

```diff
@@ -84,29 +84,29 @@ resource "cloudflare_access_group" "example" {
-- `common_name` (String)
+- `auth_method` (String) The type of authentication method. Refer to https://datatracker.ietf.org/doc/html/rfc8176#section-2 for possible types.
+- `azure` (Block List) Matches an Azure group. Requires an Azure identity provider. (see [below for nested schema](#nestedblock--include--azure))
+- `certificate` (Boolean) Matches any valid client certificate.
+- `common_name` (String) Matches a valid client certificate common name.
 - `common_names` (List of String) Overflow field if you need to have multiple common_name rules in a single policy.  Use in place of the singular common_name field.
-- `device_posture` (List of String)
-- `email` (List of String)
-- `email_domain` (List of String)
-- `email_list` (List of String)
-- `everyone` (Boolean)
-- `external_evaluation` (Block List) (see [below for nested schema](#nestedblock--include--external_evaluation))
-- `geo` (List of String)
-- `github` (Block List) (see [below for nested schema](#nestedblock--include--github))
-- `group` (List of String)
-- `gsuite` (Block List) (see [below for nested schema](#nestedblock--include--gsuite))
+- `device_posture` (List of String) The ID of a device posture integration.
+- `email` (List of String) The email of the user.
+- `email_domain` (List of String) The email domain to match.
+- `email_list` (List of String) The ID of a previously created email list.
+- `everyone` (Boolean) Matches everyone.
+- `external_evaluation` (Block List) Create Allow or Block policies which evaluate the user based on custom criteria. https://developers.cloudflare.com/learning-paths/zero-trust-web-access/advanced-workflows/external-evaluation. (see [below for nested schema](#nestedblock--include--external_evaluation))
+- `geo` (List of String) Matches a specific country.
+- `github` (Block List) Matches a Github organization. Requires a Github identity provider. (see [below for nested schema](#nestedblock--include--github))
+- `group` (List of String) The ID of a previously created Access group.
+- `gsuite` (Block List) Matches a group in Google Workspace. Requires a Google Workspace identity provider. (see [below for nested schema](#nestedblock--include--gsuite))
 - `ip` (List of String) An IPv4 or IPv6 CIDR block.
-- `ip_list` (List of String) The ID of an existing IP list to reference.
-- `login_method` (List of String)
-- `okta` (Block List) (see [below for nested schema](#nestedblock--include--okta))
-- `saml` (Block List) (see [below for nested schema](#nestedblock--include--saml))
-- `service_token` (List of String)
+- `ip_list` (List of String) The ID of a previously created IP list.
+- `login_method` (List of String) The ID of a configured Identity Provider.
+- `okta` (Block List) Matches an Okta group. Requires an Okta identity provider. (see [below for nested schema](#nestedblock--include--okta))
+- `saml` (Block List) Matches a SAML group. Requires a SAML identity provider. (see [below for nested schema](#nestedblock--include--saml))
```

Comment:
Will this work with any SAML provider, or does it need to be the Generic SAML integration (as described in dev docs)?

Reply:
saml here means the generic saml integration.